src/remote/inet.cpp

```diff
@@ -2365,9 +2359,7 @@ static bool select_wait( rem_port* main_port, Select* selct)
 
 			// Some platforms change the timeout in the select call.
 			// Reset timeout for each iteration to avoid problems.
-			timeout.tv_sec = SELECT_TIMEOUT;
-			timeout.tv_usec = 0;
-
+			struct timeval timeout { SELECT_TIMEOUT, 0 };
```

Comment:
IMO it's not good to be order-dependend on field here.

Reply:
1) You can't know that thinking in multi-platform code
2) It maked code more difficult to read

Reply:
And done